src/test/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck has started work.");
+    }
+    @Override
+    public void stopWork() {
+        System.out.println("Truck has stopped working.");
```

Comment:
Similarly, the message 'Truck has stopped working.' could be more informative. It should clearly indicate that it is the Truck that has stopped. Consider rephrasing to 'The Truck has stopped working.' or similar.